tests/test_lab_download.py

```diff
@@ -43,7 +43,7 @@ def test_download_error(self):
                 lab.ilab,
                 [
                     "--config=DEFAULT",
-                    "download",
+                    "model" "download",
```

Comment:
Same thing here -- missing a command between "model" and "download"

Hopefully it's showing up in the current test failures too